src/components/Rubric/CriterionContainer/CriterionFeedback.test.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { shallow } from '@edx/react-unit-test-utils';
+import { feedbackRequirement } from 'data/services/lms/constants';
+
+import CriterionFeedback from './CriterionFeedback';
+
+describe('<CriterionFeedback />', () => {
+  const props = {
+    criterion: {
+      feedbackValue: 'feedback-1',
+      feedbackIsInvalid: false,
+      feedbackOnChange: jest.fn().mockName('feedbackOnChange'),
+      feedbackEnabled: true,
+      feedbackRequired: feedbackRequirement.required,
+    },
+  };
+  describe('renders', () => {
```

Comment:
These tests should be separated by component state and should actually verify the contents, beyond the snapshots.
```
describe('render behavior', () => {
  describe('feedbackEnabled', () => {
    test('snapshot', () => {
    });
    it('renders valid form group around textArea with value and onChange from props', () => {
    });
    ...    
  });
  ...
});
```